Library/Homebrew/test/dev-cmd/audit_spec.rb

```diff
@@ -870,7 +870,6 @@ class FooAT11 < Formula
     include_examples "formulae exist", described_class::VERSIONED_KEG_ONLY_ALLOWLIST
     include_examples "formulae exist", described_class::VERSIONED_HEAD_SPEC_ALLOWLIST
     include_examples "formulae exist", described_class::PROVIDED_BY_MACOS_DEPENDS_ON_ALLOWLIST
-    include_examples "formulae exist", described_class::THROTTLED_FORMULAE.keys
```

Comment:
My understanding of this check is that it makes sure that all formulae in these lists exist in Homebrew/core. This check should still remain, but I think it makes more sense as an `audit` now. Do we have a way to add tap-wide audits? I'm thinking that running `brew audit --tap=homebrew/core` should check this, but I don't know if it needs to be checked when running `brew audit <formula>`

Reply:
I guess something similar to `audit_style` is probably right here; just a single `audit` that goes through all taps (by default or is filtered by `--tap`) and checks that their audits are appropriate.